code/game/objects/items/cards_ids.dm

```diff
@@ -682,6 +682,13 @@
 	if(loc != user)
 		to_chat(user, span_warning("You must be holding the ID to continue!"))
 		return
+	if(isnull(registered_account) || registered_account.replaceable && !registered_account.account_balance)
```

Comment:
There are like, two previous checks that assume that `registered_account` is not null before this.

Reply:
Yeah, I think it is from a time when ID cards didn't automatically get an account assigned.